gufe/tests/test_custom_json.py

```diff
@@ -223,8 +225,21 @@ def setup(self):
         ]
 
 
+def test_openff_quantity_array_roundtrip():
```

Comment:
I tried at first to put this in the existing test harness, but because it uses a combination of two codecs (FF Unit and Array) I couldn't figure out the trick.  This test exemplifies the issue well

Reply:
Yeah, this is the way to do it. The harness treats each codec as a unit (as in "unit test", not as in "unit kelvin"); this is a problem in the integration of 2 units (as in "integration test", not as in "integration over $x$" -- why does language have to be so overloaded?)